dts/bindings/syscon/syscon.yaml

```diff
@@ -10,3 +10,8 @@ include: base.yaml
 properties:
     reg:
       required: true
+    reg-io-width:
+      type: int
+      default: 4
```

Comment:
The use of `default` is discouraged. The same behaviour can be obtained by using `DT_INST_PROP_OR()` in the C implementation file.

Reply:
Done